models/engine/db_storage.py

```diff
@@ -74,3 +74,20 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """ Return a element if is found"""
+
+        if cls is not None:
```

Comment:
I think that and else condition for the CLS when is None, its necessary. to complete the requirements.